gift-certificates-app/web/src/main/java/com/epam/esm/security/JwtAuthenticationResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.esm.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class JwtAuthenticationResponse {
+
+        private int id;
```

Comment:
Could you please spacing (indent of class fields) for this file, look's like it differs from others

Reply:
done